backend/.eslintrc.cjs

```diff
@@ -25,6 +24,11 @@ module.exports = {
         ecmaVersion: "latest",
         sourceType: "module"
     },
-    plugins: ["react"],
-    rules: { "react/prop-types": "off", "import/prefer-default-export": [0] }
+    plugins: [],
+    rules: {
+        "react/prop-types": "off",
+        "import/prefer-default-export": "off",
+        "import/extensions": "off",
+        "no-console": "off"
```

Comment:
Instead of this, you can add an exception on the top of the one (or two) file where you need the console